docs/error_reporting.md

```diff
@@ -0,0 +1,19 @@
+#Error Capturing and Reporting in Ilios
+
+With the release of Ilios v3.57.0, the ability to capture and track errors and problems for real-time monitoring via the [Sentry.io](https://docs.sentry.io/) service has been added to the Ilios codebase.
+
+If your organization uses Sentry.io, you can easily enable the error-capturing functionality required for your Ilios instance to interact with the service by running the console command below in the context of your web services user (eg, `apache`, `www`, `www-data`, etc.):
```

Comment:
The sentry reporting is to us centrally so it doesn't matter what they use.   Should be something more like `If you're willing to share error reporting data with the Ilios team...`